crates/cxx-qt-lib/src/gui/qpainter.rs

```diff
@@ -189,6 +289,10 @@ mod ffi {
         #[rust_name = "set_clip_rect"]
         fn setClipRect(self: Pin<&mut QPainter>, rectangle: &QRect, operation: ClipOperation);
 
+        /// Sets the composition mode to the given mode.
```

Comment:
wonder if this warning should be somewhere?

```
Warning: Only a [QPainter](https://doc.qt.io/qt-6/qpainter.html) operating on a [QImage](https://doc.qt.io/qt-6/qimage.html) fully supports all composition modes. The RasterOp modes are supported for X11 as described in [compositionMode](https://doc.qt.io/qt-6/qpainter.html#compositionMode)().
```

and the https://doc.qt.io/qt-6/qpainter.html#CompositionMode-enum has a lot more description too